### ISSUE TITLE
Logging Links are wrong

### ISSUE THREAD

**Issue by irrationalRock** [humphd/bridge-troll#20]
So I noticed that in the logging section that it showed four different commands.
```
http://localhost:8080/?loglevel=debug
http://localhost:8080/loglevel=info
http://localhost:8080/loglevel=warn
http://localhost:8080/loglevel=error
```
I tried the first link and it works.

However, the last three don't work because they don't have the question mark before log.

I believe it should look like this:
```
http://localhost:8080/?loglevel=info
http://localhost:8080/?loglevel=warn
http://localhost:8080/?loglevel=error
```

I would be willing to correct this if you want.

**Comment by humphd** [comment]
Indeed, this is totally wrong, good find.  I'd be glad if you fixed this, yes.  Please do a PR.

**Comment by humphd** [comment]
@irrationalRock a tip for next time: use the form `fixed #20` without the word `issue` so it autocloses.  See https://help.github.com/articles/closing-issues-using-keywords/
